hack/deploy_only_cr.sh

```diff
@@ -68,8 +55,4 @@ EOF
   sleep 60
   # wait for the HCO to run with the new configurations
   ${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="1200s"
-fi
-
-${CMD} apply -n kubevirt-hyperconverged -f _out/hco.cr.yaml
-
-${CMD} wait -n "${HCO_NAMESPACE}" "${HCO_KIND}" "${HCO_RESOURCE_NAME}" --for condition=Available --timeout="30m"
```

Comment:
Why these line were removed? 

This is a critical part of deploying HCO.

Reply:
Because we are doing those things in the Go code in the functional tests. The `BeforeAll` and `AfterAll` blocks added in this PR are handling the part of labelling and un-labelling the nodes.

Reply:
These line does not change the HyperConverged CR, but creates it. It's OK to create it with the default configurations and then add the labels in the test, but it must be created.

Reply:
Can't be skipping this. Mainly for OCP.